Lookback window: explainer & validator (#941)
WICG/attribution-reporting-api: EVENT.md

If the filters match for multiple event triggers, the first matching event
trigger is used.

#### Lookback window

Lookback window is supported with an optional reserved keyword
`_lookback_window` which can be added to trigger's filters.

> Unlike other filter keys whose values must be a list of strings. The

Reviewer: ```suggestion
Unlike other filter keys whose values must be a list of strings, the
```

> `_lookback_window` key value must be a positive integer which represents a

Reviewer: ```suggestion
`_lookback_window` value must be a positive integer that represents a positive
```

duration in seconds.

When available on a filter, the duration since the source was registered must be

> smaller or equal to the parsed lookback window duration for the filter to match.

Reviewer: ```suggestion
less than or equal to the parsed lookback window duration for the filter to match.
```
